sswf-core/src/main/scala/com/bazaarvoice/sswf/service/util.scala

```diff
@@ -7,6 +7,6 @@ object util {
   // fingerprint a certain step configuration so we can be sure we are executing the step we intend to.
   // So instead of a real version number, we use the timeout. If that changes, the step is logically different.
   private[service] def stepToVersion[StepEnum <: (Enum[StepEnum] with WorkflowStep)](step: StepEnum): String =
-    s"${step.timeoutSeconds}" + ".hbtm" /*feature: consolidate timeouts to just heartbeat*/
+    s"${step.versionOverride}.${step.timeoutSeconds}"
```

Comment:
Ok @jerrylawson I've added the test (https://github.com/bazaarvoice/super-simple-workflow/pull/24/commits/5d186bd1c6e4dd169f5aeb52ce6fa2f625959d49), and it is in fact fine if you change a step's version number while a workflow is in flight.

We never parse and validate the versions; they're only used to determine if we should register the step on startup or not.